setup.py

```diff
@@ -55,6 +55,7 @@ def read_file(fname):
 
 install_requires = [
     'colorlog',
+    'CherryPy==6.0.2',
```

Comment:
So.... what determines what belongs in here vs in requirements.txt, again? Should we not just have this pull in and `readlines` from requirements.txt or something?